crates/bevy_tasks/src/usages.rs

```diff
@@ -1,9 +1,9 @@
 //! Definitions for a few common task pools that we want. Generally the determining factor for what
 //! kind of work should go in each pool is latency requirements.
 //!
-//! For CPU-intensive work (tasks that generally spin until completion) we have a standard Compute
-//! pool and an AsyncCompute pool. Work that does not need to be completed to present the next
-//! frame should go to the AsyncCompute pool
+//! For CPU-intensive work (tasks that generally spin until completion) we have a standard
+//! [`ComputeTaskPool`]  and an [`AsyncComputeTaskPool`]. Work that does not need to be completed to
+//! present the next frame should go to the [`AsyncComputeTaskPool`]
```

Comment:
Pretty sure `Compute` should be `ComputeTaskPool` and `AsyncCompute` should be `AsyncComputeTaskPool` here, so I fixed that.